markdown/ch7.markdown

```diff
@@ -150,13 +168,15 @@ res = optimize.minimize(astronaut_shift_error, 0, args=(shifted1,),
 print('The optimal shift for correction is: %f' % res.x)
 ```
 
-Brilliant! Thanks to our MSE measure, SciPy's `optimize.minimize` function has
+Brilliant! We show we shifted it by +50 pixels.
+Thanks to our MSE measure, SciPy's `optimize.minimize` function has
 recovered the correct amount to shift our distorted image to get it back to its
 original state.
 
 Unfortunately, this brings us to the principal difficulty of this kind of
 alignment: sometimes, the MSE has to get worse before it gets better. Have a
-look at the MSE value as the shift gets larger and larger: at around -300
+look at the MSE value as the shift increases: it reaches a local minimum at
+around -400, increases slightly, then at around -300
```

Comment:
I don't much like this ordering: I think of the function as starting at 0, then hitting -300, then hitting -400. What do you think of reversing the order of the narrative?